### DISCUSSION TITLE
How to map something `<leader>"` to surround without asking which charactere to use

### DISCUSSION THREAD

**Discussion by voyeg3r** [kylechui/nvim-surround#69] [Q&A]
I have tried but to no avail, which function I can require in Lua to perform this action?

I would like to map `nnoremap <leader>"` and avoid seeing: `(Surround) Motion:`

**Comment by voyeg3r** [comment]
I have two mappings that are working right now:
```
vim.keymap.set("n", "<leader>'", [[:normal ysiw'<CR>]], { desc = "surround inner word"})
vim.keymap.set('n', '<leader>"', [[:normal ysiw"<CR>]], { desc = "surround inner word"})
```

**Comment by kylechui** [comment]
My plugin doesn't have the text "Surround (Motion)"; perhaps you have a different surround plugin installed?

Edit: @voyeg3r It seems that you might have [surround.nvim](https://github.com/ur4ltz/surround.nvim/blob/633068182cf894480341b992445f0f0d2883721d/lua/surround/utils.lua#L354) installed?
